### PR TITLE
Workaround for DRF 3.4.0 compat issue

### LINKED ISSUE #156: DRF 3.4.0 ExtendedDefaultRouter compat issue
http://www.django-rest-framework.org/topics/3.4-announcement/

```
(Pdb++) self.get_api_root_view(schema_urls=urls)
*** TypeError: get_api_root_view() got an unexpected keyword argument 'schema_urls'
```

### PR DESCRIPTION
**Pull request by lwm**
Closes #156.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -13,6 +13,12 @@ setenv =
 commands=
     {envbindir}/django-admin.py test --settings=settings {posargs}
 
+[testenv:drf.3.4.0]
+deps=
+    {[testenv]deps}
+    Django>=1.10a1,<1.11
```
Comment (auvipy) [inline tox.ini]: 1.10 beta is already released

**Comment by auvipy** [comment]
a change log will also be very helpful

**Comment by lwm** [comment]
OK, I bumped the version, tagged it and manually tested the fix + moved it.

**Comment by auvipy** [comment]
i will merge this pr. if you have time then could you try to fix existing test failures?

**Comment by lwm** [comment]
21 failing tests across 9 jobs? I don't think it's gonna be me ...

**Comment by auvipy** [comment]
only the 1.8 ones will be OK.

**Comment by lwm** [comment]
In fact, this will need to be extended to accept `schema_renderers` as per [the DRF 3.4.0 docs][1].

I think we definitely need to see the relevant tests passing ... I'll work on it.

[1]: http://www.django-rest-framework.org/api-guide/schemas/#using-defaultrouter

**Comment by auvipy** [comment]
thanks you :)

**Comment by stephenemslie** [comment]
+1 This is holding me back from upgrading to drf 3.4.0.

**Comment by auvipy** [comment]
@lwm do you have anything remaining in this pr?or should i merge it?

**Comment by lwm** [comment]
@auvipy 

I have added the `**kwargs` threading in the workaround and tested it manually.

Please merge. I didn't find time to debug the rest of the unrelated problems.

**Comment by lwm** [comment]
Working on my end, @stephenemslie, can you confirm?

**Comment by stephenemslie** [comment]
Confirmed working here. Thank you!

**Comment by lwm** [comment]
:beers: :hand:

**Comment by jplock** [comment]
I just ran into this.  Any idea when a new version will be pushed to pypi?  Thanks!

**Comment by auvipy** [comment]
lots of tests are failing, if you could help to pass the 1.8 failures then there is possibility for a pypi release.

**Pull request merged**